packages/docusaurus/src/server/plugins/routeConfig.ts

```diff
@@ -42,6 +42,14 @@ export function sortConfig(
       return -1;
     }
 
+    // Wildcard also should get placed last
+    if (a.path.endsWith('/*')) {
```

Comment:
I couldn't decide between `a.path == "/*"` or `.endsWith` and decided to go with `endsWith` since it seemed less magical.